docs/deepsemgrep/deepsemgrep-introduction.md

```diff
@@ -43,12 +43,12 @@ To install DeepSemgrep, follow these steps:
     ```
 1. To test DeepSemgrep, use the following command in the root directory of the codebase to scan:
     ```bash
-    semgrep --deep --config "p/deepsemgrep" --dataflow-traces
+    semgrep --interfile --config "p/deepsemgrep" --dataflow-traces
```

Comment:
Maybe make it `--pro` to be consistent with the product name? Discussed in DMs with @milanwilliams.